t/Makefile.am

```diff
@@ -57,27 +57,32 @@ TESTS = Test01-check_programs_exist.sh \
         Test25-hydrology.sh \
         Test26-reference_fregrid_gcc.sh \
         Test27-multiple_nests.sh \
-        01-make_hgrid.sh \
+        Test28-fregrid_stretched.sh \
+        Test29-fregrid_no_stretched.sh \
+        Test30-make_coupler_mosaic_stretched_grid.sh \
+	01-make_hgrid.sh \
         01-make_vgrid.sh
 
 EXTRA_DIST = $(TESTS) \
-            Test02-input\
-            Test02-reference \
-            Test03-input \
-            Test06-input \
-            Test07-input \
-            Test08-input \
-            Test10-input \
-            Test12-input \
-            Test15-input \
-            Test16-input \
-            Test17-input \
-            Test18-input \
-            Test20-input \
-            Test17-reference \
-            Test18-reference \
-            Test20-reference \
-            $(test25_data)
+             Test02-input\
+             Test02-reference \
+             Test03-input \
+             Test06-input \
+             Test07-input \
+             Test08-input \
+             Test10-input \
+             Test12-input \
+             Test15-input \
+             Test16-input \
+             Test17-input \
+             Test18-input \
+             Test20-input \
+             Test26-input \
+             Test28-input \
+             Test30-input \
+             Test17-reference \
+             Test18-reference \
+             Test20-reference 
```

Comment:
Did you intend to remove the `$(test25_data)` from the EXTRA_DIST definition list? Also did you mean to add `Test26-input`?

Reply:
$(test25_data) is back to EXTRA_DIST definition list.